backend/src/test/java/com/huellapositiva/api/ESALMemberControllerShould.java

```diff
@@ -77,5 +83,106 @@ void register_member_and_return_201_and_tokens() throws Exception {
+                RegisterESALMemberRequestDto.builder()
+                        .name(VALID_NAME)
+                        .surname("abcd1234")
+                        .phoneNumber(VALID_PHONE)
+                        .email(DEFAULT_EMAIL)
+                        .password(DEFAULT_PASSWORD)
+                        .build(),
+                RegisterESALMemberRequestDto.builder()
+                        .name(VALID_NAME)
+                        .surname("")
+                        .phoneNumber(VALID_PHONE)
+                        .email(DEFAULT_EMAIL)
+                        .password(DEFAULT_PASSWORD)
+                        .build(),
+                RegisterESALMemberRequestDto.builder()
+                        .name(VALID_NAME)
+                        .surname(VALID_SURNAME)
+                        .phoneNumber("123987231589")
+                        .email(DEFAULT_EMAIL)
+                        .password(DEFAULT_PASSWORD)
+                        .build(),
+                RegisterESALMemberRequestDto.builder()
+                        .name(VALID_NAME)
+                        .surname(VALID_SURNAME)
+                        .phoneNumber("")
+                        .email(DEFAULT_EMAIL)
+                        .password(DEFAULT_PASSWORD)
+                        .build(),
+                RegisterESALMemberRequestDto.builder()
+                        .name(VALID_NAME)
+                        .surname(VALID_SURNAME)
+                        .phoneNumber(VALID_PHONE)
+                        .email("foo@@@huellapositiva.com")
+                        .password(DEFAULT_PASSWORD)
+                        .build(),
+                RegisterESALMemberRequestDto.builder()
+                        .name(VALID_NAME)
+                        .surname(VALID_SURNAME)
+                        .phoneNumber(VALID_PHONE)
+                        .email("   ")
+                        .password(DEFAULT_PASSWORD)
+                        .build(),
+                RegisterESALMemberRequestDto.builder()
+                        .name(VALID_NAME)
+                        .surname(VALID_SURNAME)
+                        .phoneNumber(VALID_PHONE)
+                        .email(DEFAULT_EMAIL)
+                        .password("asfas`´^][{}~~")
+                        .build(),
+                RegisterESALMemberRequestDto.builder()
+                        .name(VALID_NAME)
+                        .surname(VALID_SURNAME)
+                        .phoneNumber(VALID_PHONE)
+                        .email(DEFAULT_EMAIL)
+                        .password("asfa")
+                        .build(),
+                RegisterESALMemberRequestDto.builder()
+                        .name(VALID_NAME)
+                        .surname(VALID_SURNAME)
+                        .phoneNumber(VALID_PHONE)
+                        .email(DEFAULT_EMAIL)
+                        .password("           ")
+                        .build()
+        );
```

Comment:
Nice one.

Using VALID_XXX makes it very clear the intention and allows us to detect the invalid field we are testing immediately.

Reply:
Thank you!